cocoa/src/toga_cocoa/app.py

```diff
@@ -352,7 +352,7 @@ def show_about_dialog(self):
         if self.interface.author is None:
             options["Copyright"] = ""
         else:
-            options["Copyright"] = f"Copyright © {self.interface.author}"
```

Comment:
Why have you made this change? The copyright is a valid symbol, and it displays correctly in my testing.